stdlib/public/core/Arrays.swift.gyb

```diff
@@ -170,7 +170,7 @@ if True:
 """
   elif Self == 'Array':
     SelfDocComment = '''\
-/// An ordered, random-access collection.
+/// An ordered, random-access collection of elements of a single type.
```

Comment:
I'm going to take that back out until I think of a clearer way to express it that doesn't pretend `[Any]` doesn't exist.